gameSource/languages/help_English

```diff
@@ -11,13 +11,17 @@
 title$MY PHRASEBOOK$
 IF I SAY...
 space$0.25f
-sub$'NO BB'$
+sub$'I AM XXX.'$
+I WILL GIVE MYSELF A NAME.
+sub$'YOU ARE XXX.'$
+I WILL NAME SOMEONE.
+sub$'LET ME BE INFERTILE.'$
 I WILL NOT HAVE BABIES.
-sub$'GIMME BB'$
+sub$'LET ME BE FERTILE.'$
 I WILL HAVE BABIES AGAIN.
-sub$'I SAY SOMETHING'$
+sub$'I SAY XXX'$
 I WILL WHISPER A PASSWORD.
-sub$'MAGIC WORD IS SOMETHING'$
+sub$'MAGIC WORD IS XXX'$
```

Comment:
Again, what difference does this change make?
Please also make sure to mention this change in a commit message in future.

Reply:
just standardising. XXX instead of SOMEONE, SOMETHING, SOME OBJECT, ect